src/Microsoft.DotNet.Interactive.Formatting/Formatter.cs

```diff
@@ -106,10 +107,14 @@ public static void ResetToDefault()
             _preferredMimeTypes.Clear();
             _defaultTypeFormatters.Clear();
             _defaultPreferredMimeTypes.Clear();
-            _defaultTypeFormatters.PushRange(HtmlFormatter.DefaultFormatters);
-            _defaultTypeFormatters.PushRange(JsonFormatter.DefaultFormatters);
-            _defaultTypeFormatters.PushRange(PlainTextFormatter.DefaultFormatters);
-
+            
+            // In the lists of default formatters, the highest priority ones come first,
+            // so register those last.
+            _defaultTypeFormatters.PushRange(HtmlFormatter.DefaultFormatters.Reverse().ToArray());
+            _defaultTypeFormatters.PushRange(JsonFormatter.DefaultFormatters.Reverse().ToArray());
+            _defaultTypeFormatters.PushRange(PlainTextFormatter.DefaultFormatters.Reverse().ToArray());
+
+            // It is unclear if we need this default:
             _defaultPreferredMimeTypes.Push((typeof(string), PlainTextFormatter.MimeType));
```

Comment:
I'm not sure we need this default mime type for `string` anymore. Maybe there's a reason why